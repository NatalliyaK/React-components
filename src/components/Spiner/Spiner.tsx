import React, { Component } from 'react';
import './Spiner.css';

export default class Spiner extends Component {
  render() {
    return (
      <div className="lds-ellipsis">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
      </div>
    );
  }
}
