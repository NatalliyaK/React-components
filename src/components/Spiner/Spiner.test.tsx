import { render } from '@testing-library/react';
import Spiner from './Spiner';

describe('Spiner component', () => {
  it('render correctly', () => {
    const { asFragment } = render(<Spiner />);

    expect(asFragment()).toMatchSnapshot();
  });
});
