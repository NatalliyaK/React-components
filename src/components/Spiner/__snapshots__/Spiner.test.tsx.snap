// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardsPage component render correctly 1`] = `
<DocumentFragment>
  <div
    class="lds-ellipsis"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</DocumentFragment>
`;

exports[`Spiner component render correctly 1`] = `
<DocumentFragment>
  <div
    class="lds-ellipsis"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</DocumentFragment>
`;
